client/state/data-layer/wpcom/sites/plugins/recommended/test/index.js

```diff
@@ -0,0 +1,65 @@
+/** @format */
```

Comment:
These days we would usually refrain from writing such tests, as all they do is duplicate the code they test, and that really adds more overhead than value.

Reply:
Good to know - I'll go ahead and scrap this file.